src/sql/base/browser/ui/infoBox/infoBox.ts

```diff
@@ -96,6 +127,51 @@ export class InfoBox extends Disposable implements IThemable {
 		}
 	}
 
+	public get isClickable(): boolean {
+		return this._isClickable;
+	}
+
+	public set isClickable(v: boolean) {
+		this._isClickable = v;
```

Comment:
Return if v and the current value of _isClickable are the same - otherwise you could end up registering duplicate listeners if this is called multiple times.